src/OrchardCore.Modules/OrchardCore.Liquid/Drivers/LiquidPartDisplay.cs

```diff
@@ -10,17 +11,16 @@ namespace OrchardCore.Liquid.Drivers
 {
     public class LiquidPartDisplay : ContentPartDisplayDriver<LiquidPart>
     {
-        private readonly IContentDefinitionManager _contentDefinitionManager;
         private readonly ILiquidTemplateManager _liquidTemplatemanager;
 
-        public LiquidPartDisplay(
-            IContentDefinitionManager contentDefinitionManager,
-            ILiquidTemplateManager liquidTemplatemanager)
+        public LiquidPartDisplay(ILiquidTemplateManager liquidTemplatemanager, IStringLocalizer<LiquidPartDisplay> localizer)
         {
-            _contentDefinitionManager = contentDefinitionManager;
             _liquidTemplatemanager = liquidTemplatemanager;
+            T = localizer;
         }
 
+        public IStringLocalizer T { get; private set; }
```

Comment:
Okay

Note: it is done like this in 5 other places in the source code ;)

Reply:
I will clean up everything, I don't think it's suitable anymore while the value set within the constructor

Reply:
Yes, it's only set in the ctor and only for internal usage so we don't need a setter. I think we made it a property just to use an Uppercase `T` without breaking the naming convention.

I already removed in some places the `private` access modifier, not sure it's good but it is defined like this in most other places.

    public IStringLocalizer T { get; set; }

Oh yes, maybe you meant to remove the setter completely which makes more sense.

    public IStringLocalizer T { get; }

Or just this but that breaks the naming convention, but it doesn't hurt me, i already saw it in the aspnetcore code.

    private IStringLocalizer T;

**So i let you cleanup this as you want in another PR ;)**

But but but, maybe this is not the end of the story. As i remember for all `IStringLocalizer` it would be better to use `S` in place of `T`. And for all `IHtmlLocalizer` it would be better to use `H`.